### ISSUE TITLE
Version 1.8.4 introduced a breaking of layout for iPhones with SafeArea

### ISSUE THREAD

**Issue by Ariandr** [kevin0571/STPopup#125]
I believe it's connected to the fix https://github.com/kevin0571/STPopup/issues/102

It broke my layout, because now the root controller doesn't stick to the bottom and it creates a gap within the safe area. I believe it shouldn't be like that and it should be our job to control the layout of the root controller, because now it seems we can't stick it back to the bottom of the screen.

I also think it might have broken the behavior of many apps, so, please consider to revert it or provide a way to stick it to the bottom. If you know the way to do that by just using some properties, please let me know.

Thanks for the great library! Hope you can help me.

```
popController = STPopupController(rootViewController: myViewController)
let blurEffect = UIBlurEffect(style: .dark)
popController?.backgroundView = UIVisualEffectView(effect: blurEffect)
popController?.containerView.backgroundColor = UIColor.clear
popController?.style = STPopupStyle.bottomSheet
popController?.navigationBarHidden = true
popController?.present(in: self)
```
<img width="536" alt="v_1_8_4" src="https://user-images.githubusercontent.com/13591333/54428781-35989700-4727-11e9-90eb-19f05de570ed.png">

<img width="536" alt="Screen Shot 2019-03-15 at 13 33 52" src="https://user-images.githubusercontent.com/13591333/54428757-20bc0380-4727-11e9-9dfa-3cc24f794032.png">

**Comment by kevin0571** [comment]
`safeAreaInsets` is exposed in STPopupController in 1.8.5.
Set a 0 insets should avoid the gap.

**Comment by Ariandr** [comment]
@kevin0571 Thanks for the quick fix! Appreciate it :)

**Comment by Ariandr** [comment]
Hi @kevin0571 
It seems you didn't push changes for the release 1.8.5. The latest changes are those ones which were done for 1.8.4.

 And I updated the pod, but `safeAreaInsets` isn't exposed.

<img width="475" alt="Screenshot at Mar 18 12-56-50" src="https://user-images.githubusercontent.com/13591333/54525832-bbae1b00-497d-11e9-8890-098c1316c77b.png">
<img width="515" alt="Screenshot at Mar 18 12-56-26" src="https://user-images.githubusercontent.com/13591333/54525889-d8e2e980-497d-11e9-8473-7ad2342443cd.png">

**Comment by kevin0571** [comment]
@Ariandr I think I failed to push it without realizing it. I've re-published 1.8.6 to fix it.

**Comment by Ariandr** [comment]
@kevin0571 Thanks for quick response and fix!
